doc/reference/tt_cli/start.rst

```diff
@@ -0,0 +1,53 @@
+.. _tt-start:
+
+Starting a Tarantool instance
+=============================
+
+..  code-block:: bash
+
+    tt start INSTANCE
+
+``tt start`` starts the specified Tarantool :ref:`instance <admin-instance_file>`.
+The specified ``INSTANCE`` value is used as this instance's identifier.
+
+Details
+-------
+
+The ``INSTANCE`` argument must specify a path to the instance file:
+
+*   from the ``instances_available`` path specified in the
+    :ref:`tt configuration file <tt-config_file_app>`.
+*   from the ``tt`` :ref:`working directory <tt-config_modes>` if ``instances_available``
```

Comment:
Right now the `cwd` is default `instances_available`, but it looks like this will be changed. It would be better to remove this item.